src/LLVM_intrinsics.jl

```diff
@@ -430,10 +431,24 @@ for f in MULADD_INTRINSICS
     end
 end
 
-function fmaddsub(a::LVec{N, T}, b::LVec{N, T}, c::LVec{N, T}) where {N, T<:FloatingTypes}
-    Base.llvmcall("llvm.x86.fma.fmaddsub_pd", LVec{N, T}, (LVec{N, T}, LVec{N, T}, LVec{N, T}), a, b, c)
+
+for (t, N, T) in [("d"    , 2, Float64), ("s"    , 4, Float32),
+                  ("d.256", 4, Float64), ("s.256", 8, Float32),
+                  # ("d.512", 8, Float64), ("s.512", 16, Float32) # These don't seem supported by LLVM yet
+                 ]
+    @eval @generated function fmaddsub(a::LVec{$N, $T}, b::LVec{$N, $T}, c::LVec{$N, $T})
```

Comment:
`fmaddsub` has a simpler cousin, `faddsub`. Do you want to add support for it as well?

Reply:
The function `vfmaddsub` is sufficiently obscure that it needs documentation. Documentation in the sense of equivalent Julia code would be good; maybe something along the lines of
```Julia
fmaddsub(a,b,c) = SIMD((isodd(n) ? a[n*b[n]-c[n] : a[n]*b[n]+c[n] for n in 1:N)...)
```
Feel free to improve or modify.

Reply:
re `faddsub`: I'd be happy to add it, though I have no need for it. That being said, I don't see any documentation or commits for it anywhere, and it's definitely not in the `llvm.fma` namespace (at least, when I try to do `llvm.fma.faddsub` or `llvm.fma.vaddsub`, it doesn't recognize the call, which makes sense as it's not an `fma` call), so I'm not sure about whether it's in the scope of this PR.

re documentation: sounds great.